base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -534,19 +411,24 @@ pub fn create_lmdb_database<P: AsRef<Path>>(
         .set_env_config(config)
         .set_max_number_of_databases(15)
         .add_database(LMDB_DB_METADATA, flags)
-        .add_database(LMDB_DB_HEADERS, flags)
+        .add_database(LMDB_DB_HEADERS, flags | db::INTEGERKEY)
+        .add_database(LMDB_DB_HEADER_ACCUMULATED_DATA, flags)
+        .add_database(LMDB_DB_MMR_PEAK_DATA, flags)
         .add_database(LMDB_DB_BLOCK_HASHES, flags)
         .add_database(LMDB_DB_UTXOS, flags)
+        .add_database(LMDB_DB_INPUTS, flags)
         .add_database(LMDB_DB_STXOS, flags)
         .add_database(LMDB_DB_TXOS_HASH_TO_INDEX, flags)
         .add_database(LMDB_DB_KERNELS, flags)
         .add_database(LMDB_DB_ORPHANS, flags)
+        .add_database(LMDB_DB_ORPHAN_CHAIN_TIPS, flags)
+        .add_database(LMDB_DB_ORPHAN_PARENT_MAP_INDEX, flags | db::DUPSORT)
         .add_database(LMDB_DB_UTXO_MMR_CP_BACKEND, flags)
         .add_database(LMDB_DB_KERNEL_MMR_CP_BACKEND, flags)
```

Comment:
Perhaps remove these databases?  There are currently more than the `max_number_of_databases` setting (15)